cmd/controller/main.go

```diff
@@ -231,6 +233,36 @@ func main2() error {
 
 	go controller.Run(stop)
 
+	if !*namespaceAll && *addNamespaces != "" {
+		namespaces := strings.Split(*addNamespaces, ",")
+		var inf ssinformers.SharedInformerFactory
+		var ctlr *Controller
+
+		for _, ns := range namespaces {
+			inf = ssinformers.NewFilteredSharedInformerFactory(ssclientset, 0, ns, tweakopts)
+			ctlr = NewController(clientset, ssclientset, inf, keyRegistry)
+			ctlr.oldGCBehavior = *oldGCBehavior
+			ctlr.updateStatus = *updateStatus
+			log.Printf("Starting new informer for namespace: %s\n", ns)
+			go ctlr.Run(stop)
+		}
+	}
+
+	if !*namespaceAll && *addNamespaces != "" {
```

Comment:
hi @ThomasVitt 

With the squash, you are adding two times the same code/if condition. Could you remove it, please?